newrelic/hooks/datastore_elasticsearch.py

```diff
@@ -108,42 +108,62 @@ def _nr_wrapper_Elasticsearch_method_(wrapped, instance, args, kwargs):
 _elasticsearch_client_methods = (
     ("abort_benchmark", None),
     ("benchmark", _extract_args_index),
-    ("bulk", None),
+    ("bulk", _extract_args_index),
```

Comment:
A lot of these don't seem right to me. I believe simply replacing the `None` with `_extract_args_index` would result in different behavior that before since the `_index_name` called by `_extract_args_index` would not return `None` but `_all`. Also some of the methods have been removed which doesn't seem right either. It seems like we should probably be doing something like the following instead to maintain backwards compatibility:
```
("bulk", _extract_args_index if v8 else None),
```

Reply:
I should have explained that in the PR initially, but for some of the ones I changed, I noticed that index was still an argument in previous versions ("bulk" as an example)